package.json

```diff
@@ -62,10 +62,12 @@
   "main": "built/index.js",
   "typings": "built/index.d.ts",
   "scripts": {
+    "postinstall": "cd bin && webdriver-manager update && npm run install_website && npm run install_testapp",
```

Comment:
Yes I get it, users would not need it and not everyone is a contributor, my intention was to simplify things for contributors as its a 3 step npm install. I would remove this!